httpx/_models.py

```diff
@@ -41,7 +41,7 @@
     ResponseExtensions,
     SyncByteStream,
 )
-from ._urls import URL
+from ._urls import URL as URL
```

Comment:
I'm curious why `mypy` requires us to do this weird dance here.
Maybe we don't care why, and we just do it anyway?
Or?

Reply:
Ah gotcha.

We're importing `URL` from the wrong place, once in `_config.py` and twice in `_utils.py`. Let's fix those instead of making this change here...

```diff
--- a/httpx/_config.py
+++ b/httpx/_config.py
@@ -6,8 +6,9 @@ from pathlib import Path
 import certifi
 
 from ._compat import set_minimum_tls_version_1_2
-from ._models import URL, Headers
+from ._models import Headers
 from ._types import CertTypes, HeaderTypes, TimeoutTypes, URLTypes, VerifyTypes
+from ._urls import URL
 from ._utils import get_ca_bundle_from_env, get_logger
 
 DEFAULT_CIPHERS = ":".join(
```

And...

```diff
--- a/httpx/_utils.py
+++ b/httpx/_utils.py
@@ -16,7 +16,7 @@ import sniffio
 from ._types import PrimitiveData
 
 if typing.TYPE_CHECKING:  # pragma: no cover
-    from ._models import URL
+    from ._urls import URL
 
 
 _HTML5_FORM_ENCODING_REPLACEMENTS = {'"': "%22", "\\": "\\\\"}
@@ -465,7 +465,7 @@ class URLPattern:
     """
 
     def __init__(self, pattern: str) -> None:
-        from ._models import URL
+        from ._urls import URL
 
         if pattern and ":" not in pattern:
             raise ValueError(
```